robotcontainer.py

```diff
@@ -149,15 +151,15 @@ def configureButtonBindings(self):
             DefenseState(self.drive)
         )
 
-        ModifiableJoystickButton(self.operatorInterface.offVelocity).onTrue(
-            VelocitySetpoint(self.velocity, VelocityControl.ControlState.Off)
-        )
-        ModifiableJoystickButton(self.operatorInterface.velocitySetpoint1).onTrue(
-            VelocitySetpoint(self.velocity, VelocityControl.ControlState.Setpoint1)
-        )
-        ModifiableJoystickButton(self.operatorInterface.velocitySetpoint2).onTrue(
-            VelocitySetpoint(self.velocity, VelocityControl.ControlState.Setpoint2)
-        )
+        # ModifiableJoystickButton(self.operatorInterface.offVelocity).onTrue(
+        #     VelocitySetpoint(self.velocity, VelocityControl.ControlState.Off)
```

Comment:
Why all commented out now

Reply:
velocity setpoint not relevant to the actual lab, putting it here as comment for as-needed with testing